# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/VertexAttribute.inl
# Commit: 3a91885d47e0
# Full Hash: 3a91885d47e0ef7fba9749562d52300eea67c651
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 77cee77c0307 -r 3a91885d47e0 gfx/angle/checkout/src/libANGLE/VertexAttribute.inl
--- a/gfx/angle/checkout/src/libANGLE/VertexAttribute.inl	Fri Nov 08 16:41:42 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,58 +0,0 @@
-//
-// Copyright (c) 2015 The ANGLE Project Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-//
-// VertexAttribute.inl: Inline vertex attribute methods
-//
-
-namespace gl
-{
-
-inline VertexAttribCurrentValueData::VertexAttribCurrentValueData()
-    : Type(gl::VertexAttribType::Float)
-{
-    FloatValues[0] = 0.0f;
-    FloatValues[1] = 0.0f;
-    FloatValues[2] = 0.0f;
-    FloatValues[3] = 1.0f;
-}
-
-inline void VertexAttribCurrentValueData::setFloatValues(const GLfloat floatValues[4])
-{
-    for (unsigned int valueIndex = 0; valueIndex < 4; valueIndex++)
-    {
-        FloatValues[valueIndex] = floatValues[valueIndex];
-    }
-    Type = gl::VertexAttribType::Float;
-}
-
-inline void VertexAttribCurrentValueData::setIntValues(const GLint intValues[4])
-{
-    for (unsigned int valueIndex = 0; valueIndex < 4; valueIndex++)
-    {
-        IntValues[valueIndex] = intValues[valueIndex];
-    }
-    Type = gl::VertexAttribType::Int;
-}
-
-inline void VertexAttribCurrentValueData::setUnsignedIntValues(const GLuint unsignedIntValues[4])
-{
-    for (unsigned int valueIndex = 0; valueIndex < 4; valueIndex++)
-    {
-        UnsignedIntValues[valueIndex] = unsignedIntValues[valueIndex];
-    }
-    Type = gl::VertexAttribType::UnsignedInt;
-}
-
-inline bool operator==(const VertexAttribCurrentValueData &a, const VertexAttribCurrentValueData &b)
-{
-    return (a.Type == b.Type && memcmp(a.FloatValues, b.FloatValues, sizeof(float) * 4) == 0);
-}
-
-inline bool operator!=(const VertexAttribCurrentValueData &a, const VertexAttribCurrentValueData &b)
-{
-    return !(a == b);
-}
-
-}  // namespace gl